content/index.html

```diff
@@ -231,9 +271,9 @@ <h2>Regulatory Compliance – Federal Mandates</h2>
       <section>
         <h2>Digital Ecosystems - Identity of Transacting Entities*</h2>
         <p>
-          Verifying that a company is “authorized” is enough to meet the letter
+          Verifying that an organization is “authorized” is enough to meet the letter
           of the law; however, in digital ecosystems it is also necessary to
-          establish the <i>identity</i> of the company. Thus, it becomes a two part
+          establish the <i>identity</i> of the organization. Thus, it becomes a two part
           process – is the counterparty who they say they are, and if so, are
           they “authorized”? To add to this challenge, DSCSA transactions for PI
           Verification and TI Tracing move through any number of intermediaries
```

Comment:
Remove the line about "*Content remains under discussion. OCI expects to resolve feedback by Version 2.0 release in December 2021."